src/site/mod.rs

```diff
@@ -115,18 +105,14 @@ fn reads_description() {
 
 #[test]
 fn reads_theme() {
+    let _guard = initialize_tokio_runtime().enter();
     let site = Site::build(&config(), &config().readme_path).unwrap();
     assert!(site.html.contains("html class=\"dark\""));
 }
 
-#[test]
```

Comment:
can you say more about deleting this?

Reply:
We don't build css anymore, we just copy the files

Reply:
ah- should we add a test for that then?